awscli/topics/config-vars.rst

```diff
@@ -0,0 +1,209 @@
+:title: AWS CLI Configuration Variables
+:description: Configuration Variables for the AWS CLI
+:category: General
+:related command: configure, configure get, configure set
+
```

Comment:
We can make these pluralized. It makes more sense given that it is looking like most topics have multiple related commands and topics.